js/script.js

```diff
@@ -19,22 +25,40 @@ const addButtonHandler = () => {
   }
 };
 
+const searchButtonHandler = () => {
+  const searchTitle = sanitizeInput($searchInput.value);
+
+  if (searchTitle) {
+    const filteredTasks = tasks.filter((task) =>
+      task.title.toLowerCase().includes(searchTitle.toLowerCase())
+    );
+    if (filteredTasks.length > 0) {
+      renderTasks(filteredTasks);
+    } else {
+      showToastMessage("No tasks found matching the search.");
```

Comment:
could we keep all the messages in a utility file as constants and use it from there?

Reply:
Done, bhai.